pkg/webhook/server/server.go

```diff
@@ -21,65 +20,75 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 
 	"github.com/qinqon/kube-admission-webhook/pkg/certificate"
-	"github.com/qinqon/kube-admission-webhook/pkg/certificate/triple"
+	"github.com/qinqon/kube-admission-webhook/pkg/certificate/chain"
 )
 
 type Server struct {
-	webhookServer *webhook.Server
-	certManager   *certificate.Manager
-	log           logr.Logger
+	webhookServer  *webhook.Server
+	webhookConfigs map[certificate.WebhookReference]bool
+	certManager    func() (*certificate.Manager, error)
```

Comment:
To follow controller-runtime way https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/manager/manager.go#L227 let's call this `newCertManager` since it's a builder function.